apps/charterafrica/src/payload/blocks/Consultations.js

```diff
@@ -0,0 +1,42 @@
+import richText from "../fields/richText";
+
+const Consultations = {
+  slug: "consultations",
+  fields: [
+    richText({
+      name: "description",
+      label: {
+        en: "Description",
+        fr: "Description",
+        pt: "Descrição",
+      },
+      required: true,
+      localized: true,
+      admin: {
+        elements: [],
+        leaves: ["bold", "italic", "underline"],
+      },
+    }),
+    {
+      name: "collection",
+      label: {
+        en: "Item",
+        fr: "Article",
+        pt: "Item",
+      },
+      type: "select",
+      options: [
+        {
+          label: "Documents",
+          value: "documents",
+        },
+        {
+          label: "Youtube",
+          value: "youtube",
+        },
+      ],
+    },
```

Comment:
I think these won't be collections. Documents should be just a group from sourceAFRICA and audiovideo/multimedia will just be a YouTube playlist.

The easiest option is to have a block for each. i.e. in this PR the focus will on `ConsultationDocuments`